tests/testthat/test-conjunct_test_linter.R

```diff
@@ -1,21 +1,25 @@
 test_that("conjunct_test_linter skips allowed usages of expect_true", {
-  expect_lint("expect_true(x)", NULL, conjunct_test_linter())
-  expect_lint("testthat::expect_true(x, y, z)", NULL, conjunct_test_linter())
+  linter <- conjunct_test_linter()
+
+  expect_no_lint("expect_true(x)", linter)
```

Comment:
Maybe we should do a cleanup of `expect_lint(., NULL, .)` to avoid these small intermixed diffs. WDYT?

Reply:
Not sure there's an ideal approach since there's 100s of call sites still -- the change here is consistent with the original discussion though, to make these edits incrementally:

https://github.com/r-lib/lintr/issues/2737#issuecomment-2657481040